src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import FormCheck from 'react-bootstrap/FormCheck'
+
+interface Props {
+  /** The id for the switch element */
+  id: string
+  /** The label to render next to the switch */
+  label: string
+  /**Gives the switch a name */
+  name?: string
```

Comment:
Is this property useful in the FormCheck component? I can't find it in the documentation

Reply:
@jackcmeyer

Reply:
Hmmm... I think it probably can be removed

Reply:
Are we going to remove this?

Reply:
@irvelervel ?

Reply:
@fox1t @irvelervel I have removed it :)

Reply:
👍